terraform/environments/data-platform/api.tf

```diff
@@ -97,38 +99,54 @@ resource "aws_api_gateway_resource" "data_product_name" {
   rest_api_id = aws_api_gateway_rest_api.data_platform.id
 }
 
-# /data-product/{data-product-name}/ingest resource
-resource "aws_api_gateway_resource" "ingest_data_for_data_product" {
+# /data-product/{data-product-name}/table resource
+resource "aws_api_gateway_resource" "data_product_table" {
   parent_id   = aws_api_gateway_resource.data_product_name.id
-  path_part   = "ingest"
+  path_part   = "table"
+  rest_api_id = aws_api_gateway_rest_api.data_platform.id
+}
+
+# /data-product/{data-product-name}/table/{table-name} resource
+resource "aws_api_gateway_resource" "data_product_table_name" {
+  parent_id   = aws_api_gateway_resource.data_product_table.id
+  path_part   = "{table-name}"
+  rest_api_id = aws_api_gateway_rest_api.data_platform.id
+}
+
+# /data-product/{data-product-name}/table/{table-name}/upload resource
+resource "aws_api_gateway_resource" "upload_data_for_data_product_table_name" {
+  parent_id   = aws_api_gateway_resource.data_product_table_name.id
+  path_part   = "upload"
   rest_api_id = aws_api_gateway_rest_api.data_platform.id
 }
 
-# /data-product/{data-product-name}/ingest POST method
-resource "aws_api_gateway_method" "ingest_data_for_data_product" {
+# /data-product/{data-product-name}/table/{table-name}/upload POST method
```

Comment:
Did we decide to go with upload vs ingest? I thought ingest was quite a good name since it's kicking off a multistep process